tests/unit/elements/ImageView.spec.js

```diff
@@ -0,0 +1,108 @@
+import { shallowMount } from '@vue/test-utils'
+
+import ImageView from '@/elements/ImageView/ImageView'
+
+describe('ImageView.vue', () => {
+  const SOURCE = 'Source Demo'
+  const ALTERNATE_TEXT = 'Hello'
+  const TITLE = 'Title Demo'
+  const PRIMARY_DIMENSION_HEIGHT = 'height'
+  const PRIMARY_DIMENSION_WIDTH = 'width'
+  const IS_CENTERED = true
+  const IS_HOVERABLE = true
+  const SIZE = 'mega'
+  const SLOT = '<strong>Hello World</strong>'
+  let defaultConfig = null
+
+  beforeEach(() => {
+    defaultConfig = {
+      propsData: {
+
+        source: SOURCE,
+        alternateText: ALTERNATE_TEXT
+
+      }
+    }
+  })
+
+  function getWrapper () {
+    return shallowMount(ImageView, defaultConfig)
+  }
+
+  it('renders the correct image for the given source', () => {
+    const wrapper = getWrapper()
+
+    expect(wrapper.find(`img[src="${SOURCE}"]`).exists()).toBe(true)
+  })
+
+  it('renders the correct alternate text', () => {
+    const wrapper = getWrapper()
+
+    expect(wrapper.find(`img[alt="${ALTERNATE_TEXT}"]`).exists()).toBe(true)
+  })
+
+  it('renders the correct title', () => {
+    defaultConfig.propsData.title = TITLE
+    defaultConfig.propsData.isHoverable = IS_HOVERABLE
+    const wrapper = getWrapper()
+
```

Comment:
Remove all these newlines or alternatively leave newlines in all the tests
Consistency is important

Reply:
Solved.